design-documents/graph-ql/coverage/payflowpro-vault.md

```diff
@@ -0,0 +1,32 @@
+# Overview
+
+In the scope of extending the GraphQL coverage, we need to add Vault support to PayPal Payflow Pro payment integration.
+
+## Mutations
+
+We need to extend existing `PayflowProInput` by adding a possibility to store a Vault token:
+
+```graphql
+input PayflowProInput @doc(description:"Required input for Payflow Pro and Payments Pro payment methods.") {
+    cc_details: CreditCardDetailsInput! @doc(description: "Required input for credit card related information")
+    is_active_payment_token_enabler: Boolean! @doc(description:"States whether an entered by a customer credit/debit card should be tokenized for later usage. Required only if Vault is enabled for PayPal Payflow Pro payment integration.")
```

Comment:
Is this a typo? If so, please fix in the paragraph below too.

```suggestion
    is_active_payment_token_enabled: Boolean! @doc(description:"States whether an entered by a customer credit/debit card should be tokenized for later usage. Required only if Vault is enabled for PayPal Payflow Pro payment integration.")
```

Reply:
No, it's not a typo. We have the same name for all existing implementations in Luma and Braintree Vault GraphQL uses the same name.